caas/broker.go

```diff
@@ -14,8 +14,17 @@ type Broker interface {
 	// a charm for the specified application.
 	EnsureOperator(appName, agentPath string, config *OperatorConfig) error
 
+	// EnsureService creates or updates a service for pods with the given spec.
+	EnsureService(appName, unitSpec string, numUnits int32, config ResourceConfig) error
```

Comment:
any particular reason to use int32 and not int here? seems a bit unusual for an API at this level

Reply:
The underlying k8 api uses int32, but I can change to int here